packages/cli/src/build/index.ts

```diff
@@ -48,9 +48,6 @@ function writeOutputFile(
           outputFile.map.toString()
         );
       }
-      if (outputOptions.sourcemap !== "hidden") {
```

Comment:
it seems that they now emit this on their own so we don't have to append it manually